app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,18 @@
+class LikesController < ApplicationController
+  def create
+    @like = current_user.likes.new(like_params)
+    @post = Post.find(params[:like][:post_id])
```

Comment:
- [ ] [OPTIONAL] everything is functioning perfectly 👍. remember, you can still use the strong parameters `comment_params` that are already validated to get the `post_id`.
```suggestion
    @post = Post.find(like_params[:post_id])
```